tests/Jenkins/dirac_ci.sh

```diff
@@ -93,8 +93,6 @@ source "${TESTCODE}/DIRAC/tests/Jenkins/utilities.sh"
 installSite() {
   echo "==> [installSite]"
 
-  prepareForServer
```

Comment:
why moving it here ?

Reply:
this is needed for py2 installations only